README.md

```diff
@@ -1,2 +1,5 @@
 # frontend-boilerplate
-Want to create a React app? Start here!
+Want to create a frontend application? Start here! You can find boilerplate and explanations for
+how they work in each subdirectory of this repository.
+
```

Comment:
Thoughts on adding a line about the different subdirectory and it's for?

i.e.
`react-redux-boilerplate`:  Simple app using react.  Uses redux for store management and Clever components.  Does not handle navigation or routing.